packages/ui/lib/src/widgets/media/image.dart

```diff
@@ -49,13 +59,17 @@ class _LMImageState extends State<LMFeedImage> {
 
   @override
   Widget build(BuildContext context) {
-    style = widget.style ?? LMFeedTheme.of(context).mediaStyle.imageStyle;
+    style = widget.style ?? LMFeedTheme.instance.theme.mediaStyle.imageStyle;
     return GestureDetector(
       onTap: () => widget.onMediaTap?.call(),
       child: widget.imageUrl != null
-          ? ClipRRect(
```

Comment:
This was needed to properly round the image, have we tested if image still gets borderRadius applied?